Core/Tooltip.lua

```diff
@@ -8,7 +8,7 @@ local class = L.classBits[select(2, UnitClass("PLAYER"))];
 
 
 --// Tooltip
-mog.tooltip = CreateFrame("Frame", "MogItTooltip", UIParent, "TooltipBorderedFrameTemplate");
+mog.tooltip = CreateFrame("GameTooltip", "MogItTooltip", UIParent, "GameTooltipTemplate");
```

Comment:
Is this actually necessary? Despite the naming this frame is not actually a tooltip.